feature/gribi/otg_tests/basic_encap_test/basic_encap_test.go

```diff
@@ -674,6 +672,71 @@ func configureBaseconfig(t *testing.T, dut *ondatra.DUTDevice) {
 	gnmi.Replace(t, dut, gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).PolicyForwarding().Config(), pf)
 }
 
+func staticARPWithMagicUniversalIP(t *testing.T, dut *ondatra.DUTDevice) {
+	t.Helper()
+	p2 := dut.Port(t, "port2")
+	p3 := dut.Port(t, "port3")
+	p4 := dut.Port(t, "port4")
+	p5 := dut.Port(t, "port5")
+	s2 := &oc.NetworkInstance_Protocol_Static{
+		Prefix: ygot.String(magicIp + "/32"),
+		NextHop: map[string]*oc.NetworkInstance_Protocol_Static_NextHop{
+			strconv.Itoa(nh10ID): {
+				Index: ygot.String(strconv.Itoa(nh1ID)),
```

Comment:
I think it should be nh10ID instead of nh1ID

Reply:
I have fixed this function to use a range index for the OC static route config instead of the corresponding nh AFT index to simplify the code